### PR TITLE
Misc HTML doc generation changes

### PR DESCRIPTION
**Pull request by kianmeng**
List of changes:

- Add link to GitHub in function header
- Fix code indentation
- Add spacing to badges
- Comment format

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,30 +50,30 @@ role **roles/iam.serviceAccountTokenCreator** on `GSA_SIGNER`.
...
- GcsSignedUrl.generate(client, "my-bucket", "my-object.mp4")
- GcsSignedUrl.generate(client, "my-bucket", "my-object.mp4", expires: GcsSignedUrl.hours_after(3))
- ```
+1.  Load the client
+    ```elixir
+    iex> GcsSignedUrl.Client.load_from_file("/home/alexandrubagu/config/google.json")
+    ```
+or
+
+    ```elixir
```
Comment (mruoss) [inline README.md:70]: This does not render well... see the rich diff!

**Review by mruoss — COMMENTED** [review]
Thanks for your pull request. There is one little issue. Also please add an entry to the CHANGELOG, thanks.

**Comment by kianmeng** [comment]
> Thanks for your pull request. There is one little issue. Also please add an entry to the CHANGELOG, thanks.

Done.

File: README.md (hunk 1 above)
Comment (kianmeng) [reply README.md:71]: > This does not render well... see the rich diff!

See if this still an issue?

Through VsCode.
image

Through `ex_doc`.
image

**Pull request merged**